src/lib.rs

```diff
@@ -223,10 +223,23 @@ pub fn binary_response<T>(status_code: T, body: Vec<u8>) -> Response
     http::response::Builder::new()
         .status(status_code)
         .header(http::header::CONTENT_LENGTH, format!("{}", body.len()).as_str())
+        .header(http::header::CONTENT_TYPE, "application/octet-stream")
```

Comment:
Is this always the case? 🤔

Reply:
>  Any HTTP/1.1 message containing an entity-body SHOULD include a
   Content-Type header field defining the media type of that body. If
   and only if the media type is not given by a Content-Type field, the
   recipient MAY attempt to guess the media type via inspection of its
   content and/or the name extension(s) of the URI used to identify the
   resource. If the media type remains unknown, the recipient SHOULD
   treat it as type "application/octet-stream".

(quoted from RFC 2616, 7.2.1 Type)

I think, it should be "application/octet-strem" if we implement `binary_response_with_content_type`.

Reply:
thanks, I will delete it in the next commit.